spec/requests/invites_controller_spec.rb

```diff
@@ -505,6 +505,151 @@
     end
   end
 
+  describe "#create-multiple" do
+    it "fails if you are not admin" do
+      sign_in(Fabricate(:user))
+      post "/invites/create-multiple.json",
+           params: {
+             email: %w[test@example.com test1@example.com bademail],
+           }
+      expect(response.status).to eq(403)
+    end
+
+    it "creates multiple invites for multiple emails" do
+      sign_in(admin)
+      post "/invites/create-multiple.json",
+           params: {
+             email: %w[test@example.com test1@example.com bademail],
+           }
+      expect(response.status).to eq(200)
+      json = JSON(response.body)
+      expect(json["failed_invitations"].length).to eq(1)
+      expect(json["successful_invitations"].length).to eq(2)
+    end
+
+    it "creates 5000 invite codes with one request" do
+      sign_in(admin)
+      post "/invites/create-multiple.json",
+           params: {
+             email: 1.upto(5000).map { |i| "test#{i}@example.com" },
```

Comment:
change this to 2 for the test so that it doesn't slow down the test suite. Seems like a valuable test to keep for performance testing locally where can manually change the value. Remove the code comment below.